wp-content/themes/vf-wp/functions/theme.php

```diff
@@ -161,6 +161,7 @@ function vf__wp_enqueue_scripts() {
   wp_enqueue_style(
     'vfwp',
     $dir . '/assets/css/main.css',
+    $dir . '/assets/css/styles.css',
```

Comment:
I'm loading the locally built CSS here, but I've note removed the option in the admin interface to say where we want to load CSS/JS from. 

I think it would make most sense always use the theme CSS — and the admin option could become supplementary external CSS.